<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTab</class>
 <widget class="QWidget" name="SettingsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLineEdit" name="textNumberEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="emailHostLabel">
     <property name="text">
      <string>Email Host</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="emailUserEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="emailHostEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textNumberLabel">
     <property name="text">
      <string>Text Number</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="ProviderBox" name="textProviderComboBox"/>
   </item>
   <item row="10" column="0">
    <widget class="QGroupBox" name="buttons">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Save Changes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter any new values and click to</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="emailPWLabel">
     <property name="text">
      <string>Email PW</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="emailPWEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="emailUserLabel">
     <property name="text">
      <string>Email User</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="textProviderLabel">
     <property name="text">
      <string>Text Provider</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProviderBox</class>
   <extends>QComboBox</extends>
   <header>providerbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>emailUserEdit</tabstop>
  <tabstop>emailPWEdit</tabstop>
  <tabstop>emailHostEdit</tabstop>
  <tabstop>textNumberEdit</tabstop>
  <tabstop>textProviderComboBox</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>clearButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
